src/modules/common/components/RemoteContent.js

```diff
@@ -46,10 +46,14 @@ class RemoteContent extends React.Component<PropType, StateType> {
     if (!event.nativeEvent) {
       return
     }
-    const height = parseFloat(event.nativeEvent.data)
-    this.setState({
-      webViewHeight: height
-    }, this.props.onLoad)
+    const message = JSON.parse(event.nativeEvent.data)
+    if (message.type === 'error') {
+      throw Error(`An error occurred in the webview:\n${message.message}`)
```

Comment:
The error should also be sent to sentry.

Reply:
I cannot find a single instance where we explicitly send an error to sentry. 
Also sentry attaches a global listener to uncaught exceptions and unhandled promise rejections (see https://docs.sentry.io/platforms/javascript/). So we shouldn't need to tell sentry to track the error explicitly... What do you think?

Reply:
You are right, these errors are cought by sentry. This also crashes the app though. Should we only log an error and send it to sentry? (`console.error(); Sentry.log()`)

Reply:
No, it doesn't crash the app. since the error is not thrown within a react lifecycle method.